programs/nft-voter/src/state/registrar.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    error::NftVoterError,
+    id,
+    state::CollectionConfig,
+    tools::{spl_token::get_spl_token_amount, token_metadata::get_token_metadata_for_mint},
+};
+use anchor_lang::prelude::*;
+use spl_governance::tools::spl_token::{get_spl_token_mint, get_spl_token_owner};
+
+/// Registrar which stores NFT voting configuration for the given Realm
+#[account]
```

Comment:
nitpick: did you think about using zero copy?

Reply:
It's not clear to me when we should use zero copy. The documentation says it's experimental feature and should only be used when needed for performance. Is it still the case or this should be the default choice?